test/__tests/twoway.js

```diff
@@ -749,3 +749,21 @@ if ( hasUsableConsole ) {
 		console.warn = warn;
 	});
 }
+
+test( 'Contenteditable works with lazy: true (#1933)', t => {
+	const ractive = new Ractive({
+		el: fixture,
+		template: '<div contenteditable="true" value="{{value}}"></div>',
+		lazy: true
+	});
+
+	const div = ractive.find( 'div' );
+	div.innerHTML = 'foo';
+
```

Comment:
sorry late to the party, but shouldn't we test `t.ok( !ractive.get( 'value' ) )` here, otherwise test would work lazy true or false.